### PR TITLE
added check 'service schema` as a pointer

### PR DESCRIPTION
**Pull request by vpashka**
Hello. 
What do you think about this addition?

### CODE REVIEW

File: broker/broker.go [hunk 1]
```diff
@@ -201,6 +201,15 @@ func (broker *ServiceBroker) addService(svc *service.Service) {
 // createService create a new service instance, from a struct or a schema :)
 func (broker *ServiceBroker) createService(svc interface{}) (*service.Service, error) {
 	schema, isSchema := svc.(moleculer.ServiceSchema)
+
```
Comment (pentateu) [inline broker/broker.go]: Hi @vpashka ,

Thank you for the interest in moleculer go and for you contribution :)

This is good.

My only recommendation is to move the whole logic to a separate function.
E.g
```
func (broker *ServiceBroker) resolveSchema(svc interface{}) (moleculer.ServiceSchema, bool) {
  schema, isSchema := svc.(moleculer.ServiceSchema)
  if !isSchema {
		s, ok := svc.(*moleculer.ServiceSchema)
		if ok {
			schema = *s
			isSchema = ok
		}
	}

  return schema, isSchema
}
```

File: broker/broker_test.go [hunk 2]
```diff
@@ -157,4 +157,31 @@ var _ = Describe("Broker", func() {
 		Expect(result.Value()).Should(Equal(actionResult))
 	})
 
+	It("Should make a local call and return results (service schema as pointer)", func() {
```
Comment (pentateu) [inline broker/broker_test.go]: Awesome!

File: broker/broker.go (hunk 1 above)
Comment (vpashka) [reply broker/broker.go]: Done.. )

**Comment by pentateu** [comment]
Thank you.

**Comment by vpashka** [comment]
Thank you for your project!

**Pull request merged**